styles.css

```diff
@@ -0,0 +1,88 @@
+.menu-logo div {
+  width: 25px;
+  height: 3px;
+  background-color: #6070ff;
+  margin: 4px 0;
+}
+
+.logo {
+  font-family: 'Poppins', "Times New Roman", Times, serif;
```

Comment:
- [ ] It appears that the chosen font-family **"Poppins"** is not rendering correctly for some reason. Please investigate and resolve the issue to ensure the desired font is displayed properly. 😉

Reply:
issue with font family has been resolved.